docs/pallas/tpu/matmul.ipynb

```diff
@@ -674,7 +674,7 @@
     "\n",
     "### Fused right-hand-side transpose\n",
     "\n",
-    "A common first thing to do is to fuse a transpose. What do we mean by that? Suppose we wanted to compute `x @ y.T` instead of `x @ y`. Naively we could first compute `y.T` and then pass it into our efficient matrix multiply kernel. However, the operation `y.T` is not free on its own -- it involves copying `O(n^2)` data. Ideally, we could compute the transpose *while* doing the matrix multiply in just one kernel, i.e. \"fusing\" it with the matmul. \n",
```

Comment:
Maybe break this into multiple lines?

Reply:
All paragraphs in the tutorial are single lines AFAICT. So, if we want to break them into multiple lines, we probably need to do that to all of them (and in a separate PR?).